src/screens/OtherProfile/index.js

```diff
@@ -439,7 +442,8 @@ const OtherProfile = () => {
     };
     const result = await setFollow(data);
     if (result.code === 200) {
-      fetchOtherProfile();
+      handleUpdateDiscoveryUser(other_id, true);
+      fetchOtherProfile(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `fetchOtherProfile` function is now called with `true` as an argument when following a user. Again, ensure that this is the intended behavior.
```diff
- fetchOtherProfile();
+ fetchOtherProfile(true);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->